app/view_models/workarea/storefront/product_view_model/cache_key.decorator

```diff
@@ -4,9 +4,9 @@ module Workarea
 
     def option_parts
       option = @product.browse_option
-      return super unless option.present? && @options[option].present?
+      return super unless option.present? && @options['option'].present?
```

Comment:
this won't work when the ProductViewModel is used in search results. we probably need both the original code and your new modifications, unfortunately...